Modelica/Resources/Scripts/Dymola/Mechanics/MultiBody/Examples/Systems/Run.mos

```diff
@@ -1,5 +1,5 @@
 
-simulateModel("Modelica.Mechanics.MultiBody.Examples.Systems.RobotR3.fullRobot", stopTime=2, method="dassl", resultFile="fullRobot");
+simulateModel("Modelica.Mechanics.MultiBody.Examples.Systems.RobotR3.FullRobot", stopTime=2, method="dassl", resultFile="fullRobot");
```

Comment:
Shall it additionally be `resultFile="FullRobot"`?

Reply:
> 
> 
> Shall it additionally be `resultFile="FullRobot"`?

Right.

Reply:
No need to change the file name. If you do that, then you have to change `fullRobotPlot.mos`too, and users who install MSL 4.0.1 will get a new file and never overwrite the old file which will be lying around for ever (on Linux). I suggest keeping the old name.

Reply:
Actually, there shall be changed `Run.mos` as well since it calls the model too.

Reply:
Note that line 18 is problematic here as well.